Directory.Packages.props

```diff
@@ -0,0 +1,26 @@
+﻿<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="System.Memory" Version="4.5.5" />
+    <PackageVersion Include="FastExpressionCompiler.Internal.src" Version="4.2.0" />
+    
+    <!-- Benchmarks -->
+    <PackageVersion Include="BenchmarkDotNet" Version="0.13.12" />
+    <PackageVersion Include="Pidgin" Version="3.2.3" />
+    <PackageVersion Include="Sprache" Version="2.3.1" />
+    <PackageVersion Include="Superpower" Version="3.0.0" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+  
+    <!-- Global Package References -->
+    <GlobalPackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="all" />
```

Comment:
` PrivateAssets="all" ` probably has no effect here

Reply:
"here", do you mean because Microsoft.SourceLink.GitHub doesn't need it? `GlobalPackageReference` actually maintains it, had to learn since you taught me that it didn't have any effect in the PackageVersion tags. https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management#global-package-references

Reply:
And double-checked, SourceLink requires the PrivateAssets atribute (was obvious)

Reply:
Well from Jint's viewpoint...

Deps in Packages.props:

https://github.com/sebastienros/jint/blob/dadeb15519afddcb158cbe3cf83a8be9a393d377/Directory.Packages.props#L31-L35

No deps on actual artifact:

https://www.nuget.org/packages/Jint#dependencies-body-tab

Reply:
I think I found the explanation:

> Source Link package is a development dependency, which means it is only used during build. It is therefore recommended to set PrivateAssets to all on the package reference. This prevents consuming projects of your nuget package from attempting to install Source Link.
Referencing any Source Link package in a .NET SDK 8+ project suppresses Source Link that is included in the SDK.

Reply:
Well, it's actually saying that it's by default in 8.0 SDK or something like that.

Reply:
Yes, generally any `<GlobalPackageReference>` is kind of a fire-and-forget, it won't/shouldn't be present in actual package

Reply:
A reason I like to have them in a separate `<ItemGroup>` as they behave differently

Reply:
Keeping it, for safety

Reply:
https://grep.app/search?q=GlobalPackageReference%20Include%3D%22Microsoft.SourceLink.GitHub%22&filter[repo.pattern][0]=microsoft

Like you said. I asked internally to understand how it's done and what is the recommendation.

Reply:
I only understand your comment now, fixing ...